week-1/interpret/percentage-change.js

```diff
@@ -9,12 +9,32 @@ const percentageChange = (priceDifference / carPrice) * 100;
 
 console.log(`The percentage change is ${percentageChange}`);
 
+
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
 
+<<<<<<< HEAD
+// There are 3. Line 4, Line 7 and Line 10.
```

Comment:
This is incorrect.  There is no function call on line 7.  To identify function calls you need to look for the name and parenthesis, ie `func(...)` which indicate this is a function.  So there are TWO functions on each of lines 4 and five.  The functions are `Number(..)` and `replaceAll(...)`.  Line 10 has a function call `console.log(...)`.  In total there are five function calls